Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -391,14 +393,15 @@ static OSStatus sampleCallback(CMBufferRef buffer, void* refcon)
         return;
     }
 
-    ++m_framesBeingDecoded;
+    m_compressedSampleQueueSize++;
```

Comment:
Don't we usually ++a or --b instead of c++?

Reply:
I don't think it's a rule; I personally never use ++a , but I don't mind changing